bin/removeNPMAbsolutePaths

```diff
@@ -14,11 +14,17 @@ const args = process.argv.slice(2);
 const folder = args[0];
 const opts = {
   force: false,
+  userSpecifiedKeys: false,
```

Comment:
`userSpecifiedKeys` is not really necessary. You can just check if `keys` is defined (and rename it to `fields` for clarity).